### PR TITLE
Take two. New drinks

### PR DESCRIPTION
**Pull request by Mewchild**
Now that I've gotten sleep in me, and tested (call me an idiot for not testing before committing), things are good to go on this. Adds several new drinks to the station.

All (nonmilkshake) drinks with special effects are blacklisted from loadout flasks
- Bitters - 5 mint, 5 enzyme (catalyst)
- Soemmer Fire - 2 Manhattan, 1 Condenced Capsaicin, Raises body temperature
- Wine Brandy - 10 wine, 10 enzyme (catalyst). Gives 5 units
- Love Potion - 1 cream, 1 berryjuice, 1 sugar
- Morning After - 1 sbiten, 5 coffee
- Vesper - 3 gin, 1 vodka, 1 wine (follows James Bond's recipe as best as possible c;)
- Rotgut Fever Dream - 3 vodka, 1 rum, 1 whiskey, 1 cola
- Ent's Draught - 1 tonic, 1 holywater, 1 honey
- Vox Delight - 3 phoron, 1 welder fuel, 1 water. Poisons Non-Vox, heals tox damage on vox
- Screaming Viking - 1 vodka, 1 vermouth, 1 limejuice, 1 rum, 1 gin, 1 tonic
- Vile Lemon - 5 Lemonade, 1 space mountain wind
- Dream Cream - 2 milk, 1 cream, 1 honey
- Robustin - 1 antifreeze, 1 phoron, 1 welder fuel, 1 vodka
- Virgin Sip - 1 Driest Martini, 1 water
- Chocolate Milkshake - 1 milkshake, 1 coco
- Berry Shake - 1 milkshake, 1 berryjuice 
- Coffee Shake - 1 milkshake, 1 coffee. Same effects as coffee on sleepiness and drowsiness, lowers body temp.
- Jelly Shot - 4 Cherry Jelly, 1 vodka
- Named Bullet - 4 Slime Jelly, 1 vodka. Gives drinker 2u slime jelly on drink. Massive tox damage to non-promethean, heals prometheans.
- Negroni - 1 gin, 1 bitters, 1 vermouth
- Clover Club - 1 Berry Juice, 1 Lemon Juice, 3 Gin
- Old Fashioned - 3 whiskey, 1 bitters, 1 sugar
- Whiskey Sour - 2 whiskey, 1 lemon juice, 1 sugar
- Daiquiri - 3 rum, 2 limejuice, 1 sugar
- Mint Julep - 2 Whiskey, 1 water, 1 mint
- Paloma - 1 orange juice, 1 sodawater, 1 tequilla
- Mojito - 3 rum, 1 limejuice, 1 mint
- Pisco Sour - 1 Wine Brandy, 1 lemon juice, 1 sugar 
- Cold Front - 1 Iced Coffee, 1 Whiskey, 1 Mint. Lowers body temperature by a severe amount. (Functionally the opposite of antifreeze)

### CODE REVIEW

**Comment by Atermonera** [comment]
Did you... have to replace all of those tabs with sets of 4 spaces?

**Comment by Mewchild** [comment]
When I went to compile initially, I got 2000 'bad indentation' errors.

**Comment by ForFoxSake** [comment]
**Heads up!** In this PR's current state Bitters overrides the recipe for Blue Curacao, making it impossible to mix Blue Curacao, since the recipes are functionally the same.

Haven't checked fully there might be other recipes that conflict in this, I just happened to notice this one since I'd seen the Blue Curacao recipe recently.

**Comment by Mewchild** [comment]
I didn't even know it was possible to make blue curacao in game. Let me up the enzyme amount to 10 instead of five, that way you can still use orange juice for both, and choose based on how fermented you want it.

**Comment by ForFoxSake** [comment]
Changing the amounts of the reagents won't make a difference since they both require the same chemicals. It'll still eclipse one or the other, because they both require the same reagents.
I'd recommend you rework both recipes so they both require a third ingredient so that they're not competing to mix whatever recipe the game picks first.

**Comment by Mewchild** [comment]
Reasonable. Let me change it so bitters requires mint, instead of orangejuice. Still close enough to a real world type of bitters for me.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -1982,6 +1982,217 @@
...
+	result_amount = 4
+
+/datum/chemical_reaction/drinks/rotgut
+	name = "Rotgut Fever Dream"
+	id = "rotgut"
+	result = "rotgut"
+	required_reagents = list("vodka" = 3, "rum" = 1, "whiskey" = 1, "cola" = 3)
+	result_amount = 8
+
+/datum/chemical_reaction/drinks/rotgut
```
Comment (Atermonera) [inline code/modules/reagents/Chemistry-Recipes.dm]: This is defined twice

Comment (Mewchild) [reply code/modules/reagents/Chemistry-Recipes.dm]: Thank you thank you.

**Comment by Anewbe** [comment]
Can I get a complete list of drinks being added in this tossed in the OP?

**Comment by Mewchild** [comment]
I'll get it written up now, boss

**Comment by Mewchild** [comment]
Updated OP. I didn't realize how many it was until I wrote them all out in a list

**Comment by Anewbe** [comment]
Thank you much.

**Comment by Arokha** [comment]
Might want to consider a rebase without those changed tabs/spaces. Kind of silly linecount there.

**Comment by Neerti** [comment]
On this codebase we use tabs for indentation, where as this PR appears to be changing tabs in the reagent folder to spaces. Please change them back to tabs.

**Comment by Neerti** [comment]
Much better now.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 2]
```diff
@@ -1257,6 +1257,74 @@
...
+	name = "Ent's Draught"
+	id = "entdraught"
+	description = "A natural, earthy combination of all things peaceful."
+	taste_description = "fresh rain and sweet memories"
+	color = "#3a6617" // rgb(58, 102, 23)
+
+	glass_name = "Ent's Draught"
+	glass_desc = "You can almost smell the tranquility emanating from this."
+
+/datum/reagent/drink/chocoshake
```
Comment (Anewbe) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: This should probably be a child of the regular milkshake.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 3]
```diff
@@ -1257,6 +1257,74 @@
...
+	id = "chocoshake"
+	description = "A refreshing chocolate milkshake."
+	taste_description = "cold refreshing chocolate and cream"
+	color = "#8e6f44" // rgb(142, 111, 68)
+	adj_temp = -9
+
+	glass_name = "Chocolate Milkshake"
+	glass_desc = "A refreshing chocolate milkshake, just like mom used to make."
+
+/datum/reagent/drink/berryshake
```
Comment (Anewbe) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: This should probably be a child of the regular milkshake.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 4]
```diff
@@ -1257,6 +1257,74 @@
...
+	id = "Berryshake"
+	description = "A refreshing berry milkshake."
+	taste_description = "cold refreshing berries and cream"
+	color = "#ffb2b2" // rgb(255, 178, 178)
+	adj_temp = -9
+
+	glass_name = "Berry Milkshake"
+	glass_desc = "A refreshing berry milkshake, just like mom used to make."
+
+/datum/reagent/drink/coffeeshake
```
Comment (Anewbe) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: This should probably be a child of the regular milkshake.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 5]
```diff
@@ -2446,16 +2527,258 @@
...
+	strength = 10
+
+	glass_name = "Named Bullet"
+	glass_desc = "A thick slime jelly shot. You can feel your death approaching."
+
+/datum/reagent/drink/slimeshot/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
+	..()
+	if(alien == IS_DIONA)
+		return
+	M.reagents.add_reagent("slimejelly", 2)
```
Comment (Anewbe) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: I would rather this do ToxLoss than add slime jelly, or if not, add tiny amounts of slime jelly.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 6]
```diff
@@ -1982,6 +1982,212 @@
...
+
+/datum/chemical_reaction/drinks/winebrandy
+    name = "Wine brandy"
+    id = "winebrandy"
+    result = "winebrandy"
+    required_reagents = list("wine" = 10)
+    catalysts = list("enzyme" = 10) //10u enzyme so it requires more than is usually added. Stops overlap with wine recipe
+    result_amount = 5
+
+/datum/chemical_reaction/drinks/lovepotion
```
Comment (Anewbe) [inline code/modules/reagents/Chemistry-Recipes.dm]: You added a reaction for a reagent that doesn't seem to exist.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 7]
```diff
@@ -1982,6 +1982,212 @@
...
+	result_amount = 4
+
+/datum/chemical_reaction/drinks/daiquiri
+	name = "Daiquiri"
+	id = "daiquiri"
+	result = "daiquiri"
+	required_reagents = list("rum" = 3, "limejuice" = 2, "sugar" = 1)
+	result_amount = 5
+
+/datum/chemical_reaction/drinks/mintjulep
```
Comment (Anewbe) [inline code/modules/reagents/Chemistry-Recipes.dm]: You added a reaction for a reagent that doesn't seem to exist.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm (hunk 5 above)
Comment (Mewchild) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: I'll drop the slime jelly amount down to .25 of a unit. The reasoning being that it'll heal prommies, but be deadly toxic to everything else.

**Comment by Anewbe** [comment]
@Mewchild Any intention to work on this?

**Comment by Mewchild** [comment]
@Anewbe Sorry, got all the changes done except for the non-existent reagents. Life came up on my end, so I've not had much time to work on it.

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 7 above)
Comment (Mewchild) [reply code/modules/reagents/Chemistry-Recipes.dm]: I'm having the devil's time figuring this, what reagent out of these is the nonexistent one? Same for both of these.

**Comment by Anewbe** [comment]
No worries then, I'll ask again in like a week.

**Comment by Anewbe** [comment]
@Mewchild Any intention to finish this?

**Comment by Mewchild** [comment]
Yeah, I've opened it up again and looked at it. I'm having a rough time trying to figure out what reagents don't exist, but it should be good once I get that sorted.

**Comment by Atermonera** [comment]
Without looking through all of the reagents, I don't recognize honey and cocoa, and I haven't seen holy water in ages but we might still have it?

In order to properly check, you'd be best making a list of all the reagents you use in recipes, removing from that the ones that're added in this PR, then referencing the rest through the list of reagents on the github to make sure that they're all in there somewhere.

**Comment by Mewchild** [comment]
I made a point when I was making the drinks to double check that I had the correct reagent tags, which is why I'm confused on the subject.

https://github.com/PolarisSS13/Polaris/blob/42e8bb62840d1594e2a8cb9c7b952415edf1d8bf/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm#L210
https://github.com/PolarisSS13/Polaris/blob/f726e96fe78b081804f9295e72374db1f2153609/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm#L130
https://github.com/PolarisSS13/Polaris/blob/f726e96fe78b081804f9295e72374db1f2153609/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm#L88

**Comment by elgeonmb** [comment]
Honey is gettable through bee crates, which nobody orders. Coco is gettable through cocoa pods in botany. There's a flask of holy water in the Chaplain's office and any Chaplain can produce more by whacking any container full of water with a bible.

**Comment by Mewchild** [comment]
Not going to lie, I made most of these for downstream. I do know that all of the reagents exist on Polaris, however. Vorestation only has very... niche custom reagents.

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 7 above)
Comment (Anewbe) [reply code/modules/reagents/Chemistry-Recipes.dm]: Sorry for the delayed response, didn't see this. We don't have "lovepotion" or "mintjulep" reagents.

Comment (Mewchild) [reply code/modules/reagents/Chemistry-Recipes.dm]: Ah, I see. Forgot to define them in the actual chemicals list. Made the recipe without actually making the drink. I'll fix that tonight.

**Comment by Mewchild** [comment]
Got the undefined reagents fixed. Ready for review.

**Comment by OrbisAnima** [comment]
You probably will want to adjust all those to #4428 too, if it gets merged

**Pull request merged**